gwpy/signal/tests/test_filter_design.py

```diff
@@ -192,6 +192,14 @@ def test_convert_to_digital_fir(example_zpk_fs_tuple):
     assert numpy.allclose(dfilt, signal.bilinear(b, [1], fs))
 
 
+def test_convert_to_digital_complex_type_preserved():
+    z, p, k = signal.butter(3, 30, 'low', analog=True, output='zpk')
+    form, filt = filter_design.convert_to_digital((z, p, k), 100)
+    zd, pd, kd = filt
+    assert p.dtype == pd.dtype
+    assert pd.dtype == numpy.complex128
```

Comment:
I think it might be safer to use something like

```suggestion
    assert numpy.iscomplexobj(pd)
```

in case the default complex type changes in the future.